### DISCUSSION TITLE
First acceptable proposal

### DISCUSSION THREAD

**Discussion by gweobia** [strongswan/strongswan#1114] [Q&A] [answered]
In the **man** of **ipsec.conf** there is a note for **esp** parameter:

Note: As a responder, the daemon defaults to selecting the first configured proposal that's also supported by the peer. This may be changed via strongswan.conf(5) to selecting the first acceptable proposal sent by the peer instead.

How can be configured in strongswan.conf to accept **first proposal** sent by the peer as noted above?

**Comment by tobiasbrunner** [comment, marked as answer]
By disabling `charon.prefer_configured_proposals`.
